magda-db-migrator/.gitignore

```diff
@@ -0,0 +1 @@
+flyway-4.2.0/*
```

Comment:
This causes some git problems on Windows platform. I think we should delete it from git as it is third party lib.

Reply:
I have a little grief here on Windows, too, specifically with `magda-db-migrator/flyway-4.2.0/jre/lib/amd64/server/libjsig.so`. VSCode and Windows git always say this file is modified, even though git in WSL says it's unchanged. If I revert the changes in VSCode/Windows, then those environments are happy and WSL says it's modified.

This file is actually a symlink to the file with the same name in the directory above, so I assume that has something to do with it.

We can't just remove this whole directory because flyway is required to build the docker image for DB migration. Only the .tar.gz is needed, though, not the rest of the flyway distribution. There's probably another way to do it that would be better. But in the meantime, it doesn't seem to cause any actual Windows breakage, just some minor annoyance that VSCode reports a file as changed.

Reply:
OK, I will reverse the change.
I mainly use Source Tree for git operations, it always complains the libjsig.so is modified and would prevent me from doing some operations. If it only affects me, I can make the change locally.

Reply:
done